addons/account/static/src/components/many2many_tags_banks/many2many_tags_banks.js

```diff
@@ -6,6 +6,15 @@ import { registry } from "@web/core/registry";
 import { TagsList } from "@web/core/tags_list/tags_list";
 import { _t } from "@web/core/l10n/translation";
 import { onMounted } from "@odoo/owl";
+import { Many2XAutocomplete } from "@web/views/fields/relational_utils";
+
+export class Many2ManyTagsBanksAutocomplete extends Many2XAutocomplete {
+
+
+    shouldShowSearchMore(request, record) {
```

Comment:
@h4818 did something similar where we always need to show Search More, inheriting `loadOptionsSource`, shouldn't we use the same mechanism in both?
Not sure which one is the best :thinking: 
https://github.com/odoo/odoo/blob/af6117bb13f118d19cab6b7f0368bb2197e4947a/addons/account/static/src/components/many2x_tax_tags/many2x_tax_tags.js#L14-L25

Maybe @aab-odoo has an opinion?

Reply:
@william-andre @malb-odoo I think the solution introduced in this PR (with the `shouldShowSearchMore` function) it cleaner. But I agree that we'd like consistency, so it might be worth adapting many2x_tax_tags as well.